newsfragments/43902.significant.rst

```diff
@@ -0,0 +1,5 @@
+Renamed ``execution_date`` to ``logical_date`` across the codebase to align with Airflow 3.0.
+
+The shift towards ``logical_date`` helps move away from the limitations of ``execution_date``, particularly with dynamic DAG runs and cases where multiple runs occur at the same time. This change impacts database models, templates, and functions:
+- Renamed columns and function references to ``logical_date``.
+- Removed unique constraints on ``execution_date`` to support concurrent DAG runs with the same logical date.
```

Comment:
I don’t think this PR includes this second one? This was done in a previous PR and should already has its own fragment.

Reply:
Removed it.